oak_functions/examples/weather_lookup/module/src/tests.rs

```diff
@@ -127,13 +129,11 @@ fn bench_wasm_handler(bencher: &mut Bencher) {
     let lookup_data = generate_and_serialize_sparse_weather_entries(&mut rng, entry_count).unwrap();
     let lookup_data_file = oak_functions_test_utils::write_to_temp_file(&lookup_data);
 
-    let server_port = oak_functions_test_utils::free_port();
-    let server_background = oak_functions_test_utils::create_and_start_oak_functions_server(
-        server_port,
-        wasm_path_after_optimization.path().to_str().unwrap(),
-        lookup_data_file.path().to_str().unwrap(),
-    )
-    .unwrap();
+    let (_server_background, server_port) =
```

Comment:
pre-existing, but for benchmark it may make more sense to just benchmark invocations directly over the IO channel to the VM, rather than over the network (similar to #3933). i.e. it's not very useful to include the gRPC / HTTP server in the benchmark loop

Reply:
agreed, that would be better

Reply:
Tracked in #3980 3980